.travis.yml

```diff
@@ -14,6 +14,8 @@ matrix:
     env: TOX_ENV=tensorflow-2
   - python: "3.6"
     env: TOX_ENV=flake8
+  - python: "2.7"
```

Comment:
Coveralls did not accept the PR, because some py2 specific paths (e.g. [here](https://github.com/dekuenstle/sacred/blob/patch-1/sacred/observers/file_storage.py#L60)) were not tested.

Reply:
If it is just that it doesn't matter. We can still merge :D

Reply:
I reverted this commit.